WordPress/Classes/Services/ReaderPostStreamService.swift

```diff
@@ -62,10 +68,15 @@ class ReaderPostStreamService {
         do {
             let results = try context.fetch(fetchRequest)
             for object in results {
-                guard let objectData = object as? NSManagedObject else { continue }
-                context.delete(objectData)
-
-                // Checar se ta em uso ou foi salvo
```

Comment:
This translates to "_Check if it is in use or has been saved_" 😄

Reply:
Hasn't been checked until now apparently 😀